.zombienet-tests/0002-checkStorage.ts

```diff
@@ -0,0 +1,56 @@
+import { Tuple, compact, metadata } from "../packages/substrate-bindings/dist"
+import { getDynamicBuilder } from "../packages/substrate-codegen/dist"
+import { connect } from "./utils"
```

Comment:
let's move these tests into a package so there is not need to assume where are the entry points.

Then, set these as package dependencies so they are imported like

```
import { Tuple, compact, metadata } from "@polkadot-api/substrate-bindings"
import { getDynamicBuilder } from "@polkadot-api/substrate-bindings"
```

Reply:
The new package could `private` under a `tests` folder at the root

Reply:
I am not sure I understand. Creating a new package, with a package-json and adding values for avoiding a relative path (which is standard) would overcomplicate things - wouldn't it?

Reply:
The idea is to mimic actual usage and use the entry points from each package.
Actual users will do package imports.

Reply:
```
import { Tuple, compact, metadata } from "../packages/substrate-bindings/dist"
import { getDynamicBuilder } from "../packages/substrate-codegen/dist"
```
Here are tested the packages that are included inside the repository. I am not 100% sure that if I use the `@polkadot-api/..` then the pnpm will install in the CI the local packages or the versions that exist in the npm.

Reply:
This tests assumes that the package artifacts will always be in `/dist` and that there will always be a `dist/index.js`.
Those details are internal to how the package is built.

Importing the package will test
- actual user usage
- package artifacts are properly configured in the `package.json`

If you are ok, I'll configure push a commit to create the integration `test` package so `@polkadot-api/*` is properly configured for this tests.

Reply:
When I try to move the zombienet to a new package and run `yarn` I receive this:
![image](https://github.com/paritytech/polkadot-api/assets/5408605/7cf2a18b-7d93-4a32-8a85-f31c44e16d23)

I added the package (`zombie-tests`) to the `pnpm-workspace.yaml`.
My package.json is:
```json
{
  "name": "zombie-tests",
  "version": "0.0.0",
  "description": "Zombienet tests for polkadot-api",
  "main": "index.js",
  "private": true,
  "type": "module",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@types/node": "^20.8.0",
    "prettier": "^2.6.2"
  },
  "dependencies": {
    "@polkadot-api/cli": "workspace:*",
    "@polkadot-api/sc-provider": "workspace:*",
    "@polkadot-api/substrate-bindings": "workspace:*",
    "@polkadot-api/substrate-client": "workspace:*",
    "@polkadot-api/substrate-codegen": "workspace:*",
    "@polkadot-api/utils": "workspace:*",
    "@polkadot-api/client": "workspace:*",
    "typescript": "^5.2.2"
  },
  "prettier": {
    "printWidth": 80,
    "semi": false,
    "trailingComma": "all"
  }
}
```